replication-api-impl/src/test/java/com/connexta/replication/api/impl/worker/WorkerManagerTest.java

```diff
@@ -251,15 +249,14 @@ void testDestroy() {
 
   @Test
   void testNoLocalSiteConfigured() {
-    // setup
+    // given
     when(properties.getLocalSite()).thenReturn("");
-    when(siteManager.get(LOCAL_SITE)).thenThrow(NotFoundException.class);
+    when(siteManager.get("")).thenThrow(NotFoundException.class);
 
-    // when
-    new WorkerManager(siteManager, properties, threadPoolFactory);
-
-    // then
-    exception.expect(NotFoundException.class);
+    // expect
+    Assertions.assertThrows(
+        NotFoundException.class,
+        () -> new WorkerManager(siteManager, properties, threadPoolFactory));
```

Comment:
This test was not catching the exception for me so I used the junit 5 documentation to find this way of expecting an exception.